.env

```
@@ -0,0 +1,4 @@
+SATURN_NETWORK="test"
+FIL_WALLET_ADDRESS=""
+NODE_OPERATOR_EMAIL=""
+SPEEDTEST_SERVER_CONFIG=""
```

```suggestion
SPEEDTEST_SERVER_CONFIG=""
SATURN_HOME=""
```

Missing the saturn_home variable